ang/crmAttachment/attachments.html

```diff
@@ -42,5 +42,8 @@
   </div>
   <div>
     {{ts('Alternatively, you may add new files using drag/drop.')}}
+    <span ng-if="max_size">
+      {{ts('Each file must be less than %1M in size. For larger files, please upload it somewhere else and paste a link in the body of this mailing.', {1: max_size})}}
```

Comment:
I like the idea of telling them to stick it somewhere else (grin), just not sure if the suggested wording will mean anything to the average staffer. They might think it means a custom field or activity or just not know what to do. I don't have a good suggestion though without making it really long. Maybe "... upload it to Dropbox, Google Docs, etc. and paste a link ..." That's not really much better.

Reply:
have updated the msg to the above suggestion. Thanks @demeritcowboy